### PR TITLE
[567448] Fix synchronization text editor issues

### PR DESCRIPTION
**Pull request by georgiana-e**
Change-Id: Id77c85bfaf780ac1d75ee6b31418ccb19548619c
Signed-off-by: Georgiana-Elena ECOBICI <georgiana-elena.ecobici@thalesgroup.com>

### CODE REVIEW

File: plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/views/EmbeddedEditorView.java [hunk 1]
```diff
@@ -99,12 +102,17 @@ public void createPartControl(Composite parent) {
     EmbeddedEditor editor = factory.newEditor(provider).withParent(parent);
     editor.createPartialEditor();
     EmbeddedEditorInstance.setEmbeddedEditor(editor);
+    
+    // show the view
+    //eeView = (EmbeddedEditorView) activePage.showView(EmbeddedEditorView.ID);
```
Comment (claudiasima-ro) [inline plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/views/EmbeddedEditorView.java]: Remove comments if not needed anymore

File: plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/EmbeddedEditorInstance.java [hunk 2]
```diff
@@ -14,6 +14,12 @@
 
 import org.eclipse.sirius.diagram.DDiagram;
 import org.eclipse.sirius.diagram.sequence.SequenceDDiagram;
+import org.eclipse.sirius.diagram.ui.tools.api.editor.DDiagramEditor;
```
Comment (claudiasima-ro) [inline plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/EmbeddedEditorInstance.java]: Remove unused imports

File: plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/helper/EmbeddedEditorInstanceHelper.java [hunk 3]
```diff
@@ -521,4 +529,51 @@ public static String getStateFragmentName(StateFragment capellaStateFragment) {
...
     return null;
   }
+  
+
+  public static boolean isOpenedRepresentation() {
+    IWorkbenchPage activePage = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage();
+    if (activePage == null) {
+      return false;
+    }
+    for (IEditorReference ref : activePage.getEditorReferences()) {
```
Comment (claudiasima-ro) [inline plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/helper/EmbeddedEditorInstanceHelper.java:534]: Can this function make use of the following one (getOpenedRepresentation) to have less duplicated code?

**Review by claudiasima-ro — APPROVED** [review]

File: plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/views/EmbeddedEditorView.java (hunk 1 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/views/EmbeddedEditorView.java]: ok

File: plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/helper/EmbeddedEditorInstanceHelper.java (hunk 3 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/helper/EmbeddedEditorInstanceHelper.java:534]: i'll check

File: plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/EmbeddedEditorInstance.java (hunk 2 above)
Comment (georgiana-ecobici) [reply plugins/org.polarsys.capella.scenario.editor/src/org/polarsys/capella/scenario/editor/EmbeddedEditorInstance.java]: ok

**Review by georgiana-ecobici — APPROVED** [review]

**Pull request merged**